src/components/campaigns/create-form/index.tsx

```
@@ -257,33 +291,227 @@ const CampaignForm = () => {
+        // Validate Campaign Information fields
+        isValid = await trigger([
+          "name",
+          "description",
+          "category",
+          "startDate",
+          "endDate",
+          "images",
+        ] as const)
+
+        // Check specifically for images validation
+        const formImages = form.getValues("images")
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)